crates/gpu/src/gpu.rs

```diff
@@ -84,11 +86,16 @@ impl Gpu {
         let adapter_limits = adapter.limits();
         tracing::debug!("Adapter limits:\n{:#?}", adapter_limits);
 
-        #[cfg(any(target_os = "macos", target_os = "unknown"))]
-        let features = wgpu::Features::empty();
-        #[cfg(all(not(target_os = "macos"), not(target_os = "unknown")))]
-        let features =
-            wgpu::Features::MULTI_DRAW_INDIRECT | wgpu::Features::MULTI_DRAW_INDIRECT_COUNT;
+        cfg_if::cfg_if! {
+            if #[cfg(target_os = "macos")] {
+                let features = wgpu::Features::empty();
+            } else if #[cfg(target_os = "unknown")] {
+                let features = wgpu::Features::INDIRECT_FIRST_INSTANCE;
+            } else {
+                let features =
+                wgpu::Features::MULTI_DRAW_INDIRECT | wgpu::Features::MULTI_DRAW_INDIRECT_COUNT;
```

Comment:
Maybe add a comment here explaining how the renderer works in each of these cases, so that someone reading the code knows what tradeoffs are present for each architecture

Reply:
Wanted to keep this PR "small" since there is an infinite space of possibilities we could unlock.

For example at runtime configuring this feature to be off on windows as well (#191) etc.

Will do though

Reply:
Yeah no stress, a short explanation is sufficient